tests/test_evaluation_suite.py

```diff
@@ -0,0 +1,10 @@
+from unittest import TestCase
+
+from evaluate import EvaluationSuite
+
+
+class TestEvaluationSuite(TestCase):
+    def test_suite(self):
+        suite = EvaluationSuite.load("evaluate/evaluation-suite-ci")
```

Comment:
Would it make sense to just mock `evaluator` for this? Or for a separate test? That way we can test a bunch of different EvaluationSuite configs without actually running inferences, which would be useful for TDD.

Reply:
Good idea! I'd like to make sure we have at least one test which actually tests loading from a script on the Hub like this, but mocking for the rest to save compute/time.